lib/dotenv-flow.js

```diff
@@ -44,16 +44,15 @@ function cleanupDotenvDefinedVars(cwd, encoding) {
  * @param {string} cwd – path to `.env*` files directory
  * @return {string[]}
  */
-function getDotenvFilenames(cwd) {
-  const {NODE_ENV = undefined} = process.env;
+function getDotenvFilenames(cwd, node_env) {
+  const NODE_ENV = process.env.NODE_ENV || node_env;
```

Comment:
We don't need to have `const NODE_ENV` inside this function anymore, because it is now coming from the outside. And also this logic is incorrect, it gives a priority to `process.env.NODE_ENV`. Just use `node_env` as it is now provided correctly from the outside.

Reply:
got it will do

Reply:
Ok, the getDotenvFilenames Fn() has been sqauashed down to this

```
function getDotenvFilenames(cwd, node_env) {
  return [
    node_env && `${cwd}/.env.${node_env}.local`,
    node_env && `${cwd}/.env.${node_env}`,
    node_env !== 'test' && `${cwd}/.env.local`,
    `${cwd}/.env`
  ].filter(filename => filename && fs.existsSync(filename));
}
```